ridgepole.gemspec

```diff
@@ -15,11 +15,7 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
   spec.platform      = Gem::Platform::RUBY
 
-  spec.files = Dir.chdir(File.expand_path(__dir__)) do
-    `git ls-files -z`.split("\x0").reject do |f|
-      f.match(%r{\A(?:(?:test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
-    end
-  end
+  spec.files = Dir.glob('lib/**/*') + Dir.glob('bin/*') + %w[README.md LICENSE.txt CHANGELOG.md]
```

Comment:
Could you please use `git ls-files -z` to prevent including files that aren't in git?

e.g.

```ruby
  spec.files = IO.popen(%w[git ls-files -z], chdir: __dir__, err: IO::NULL) do |ls|
    ls.readlines("\x0", chomp: true).select do |f|
      (f == gemspec) ||
        f.start_with?(*%w[bin/ lib/])
    end
  end
```